src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
```

Comment:
```suggestion
  <ul className="TodoList">
```
use semantic tags, i.e `ul` for list